exercises/alphametics/HINTS.md

```diff
@@ -0,0 +1,4 @@
+## Hints
+- To parse the text, you could try to use the [FParsec](http://www.quanttec.com/fparsec/tutorial.html) library. 
+- You can solve this exercise with a brute force algorithm, but this will possibly have a poor runtime performance.
+Try to find a more sophisticated solution. Hint: You could try the column-wise addition algorithm that is usually taught in high school.
```

Comment:
I'd remove the "high" in "high school" because usually it's taught in elementary school.